serverlessworkflow/modules/ROOT/pages/tooling/serverless-logic-web-tools/serverless-logic-web-tools-deploy-projects.adoc

```diff
@@ -0,0 +1,52 @@
+= Deploying your projects
+:compat-mode!:
+// Metadata:
+:description: Serverless Logic Web Tools deploying your projects
+:keywords: kogito, workflow, serverless, editor, web, tools, settings, openshift, deploy, project
+
+Serverless Workflow projects can be deployed to an OpenShift instance via the OpenShift integration, allowing authors to test their implementations in a live environment.
```

Comment:
```suggestion
{context} projects can be deployed to an OpenShift instance via the OpenShift integration, allowing authors to test their implementations in a live environment.
```

Instead of `Serverless Workflow`, please use the attribute `{context}` instead. @hmanwani-rh I think it's a good idea to add this info in the contributions doc.